Automattic/wp-calypso
# Home: Swap photo library video for a YouTube video.

Swap out the existing photo library mp4 for a slower-paced video embedded with YouTube.

![Mar-13-2020 16-18-04](https://user-images.githubusercontent.com/1270189/76667413-71d32300-6546-11ea-80e2-f2451c1ea408.gif)

#### Testing instructions

* In developer console, type the following: `localStorage.debug='calypso:analytics*'`
* Go to My Home (`/home/:site`) and click the "Over 40,000 free images" graphic in the sidebar.
* Verify the video appears in YouTube chrome and works correctly.
* Verify that the following tracks events fire
<img width="852" alt="Screen Shot 2020-03-13 at 4 55 21 PM" src="https://user-images.githubusercontent.com/1270189/76670046-be6d2d00-654b-11ea-9e7a-517c64a14468.png">
* Click on Learn more
* Verify that we're taken to the correct support page, and that the following tracks events fires
<img width="823" alt="Screen Shot 2020-03-13 at 4 55 46 PM" src="https://user-images.githubusercontent.com/1270189/76670066-dcd32880-654b-11ea-8cb3-56928dfed3a2.png">


Fixes #39910
